bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/LedgerEntry.java

```diff
@@ -56,11 +56,9 @@
     long getLength();
 
     /**
-     * Returns the content of the entry. This method can be called only once. While using v2 wire protocol this method
-     * will automatically release the internal ByteBuf.
+     * Returns the content of the entry.
      *
      * @return the content of the entry
-     * @throws IllegalStateException if this method is called twice
      */
     byte[] getEntry();
 
```

Comment:
Returning a ByteBuf should be consistent whether using the V2 wire protocol or not. As it is now, the documentation for getEntryBuffer is super confusing.

Reply:
As in, are we returning ownership of the object? Or does the LedgerEntry own the buffer, and is this why it requires closing?

Reply:
In LedgerEntryImpl, LedgerEntry is reenterable,  closing is more for the ByteBuf release.